src/Fantomas.Tests/Integration/ConfigTests.fs

```diff
@@ -52,7 +52,7 @@ end_of_line=cr
 """
         )
 
-    let args = sprintf "%s %s" NormalVerbosity fileFixture.Filename
+    let args = sprintf "%s %s" DetailedVerbosity fileFixture.Filename
```

Comment:
We discussed we didn't want to show all exception messages unless they came from specific fantomas exceptions. However, fantomas uses `failwith` in many places instead of it's own exceptions.  Should we keep this requirement, always show all the exceptions, or rework all the places with `failwith` to throw a known Fantomas exception?